cclient/serialization/json/json_serializer.c

```diff
@@ -344,6 +344,32 @@ static retcode_t hash81_queue_to_json_array(hash81_queue_t queue,
   return RC_OK;
 }
 
+static retcode_t json_array_to_hash8019_queue(cJSON const* const obj,
+                                              char const* const obj_name,
+                                              hash8019_queue_t* queue) {
+  retcode_t ret_code = RC_OK;
+  cJSON* json_item = cJSON_GetObjectItemCaseSensitive(obj, obj_name);
+  if (cJSON_IsArray(json_item)) {
+    cJSON* current_obj = NULL;
+    cJSON_ArrayForEach(current_obj, json_item) {
+      if (current_obj->valuestring != NULL) {
+        trit_array_p tmp_ref =
+            trit_array_new_from_trytes((tryte_t*)current_obj->valuestring);
```

Comment:
Can't you just use `flex_trits_from_trytes` instead of `trit_array_new_from_trytes ` ?